diff -Nru a/setup.py b/setup.py
--- a/setup.py	2018-12-15 18:05:27.000000000 +0100
+++ b/setup.py	2018-12-16 17:26:51.522132978 +0100
@@ -28,6 +28,7 @@
 
 data_files=[('share/applications', ['data/pdfarranger.desktop']),
             ('share/pdfarranger', ['data/pdfarranger.ui']),
+            ('share/metainfo', ['data/pdfarranger.appdata.xml']),
             ('share/man/man1', ['doc/pdfarranger.1'])]
 
 setup(name='pdfarranger',
